services/ui-src/src/components/cards/ObjectiveProgressEntity.tsx

```diff
@@ -55,17 +54,45 @@ const wereTargetsMetForObjectiveProgress = (
   );
 };
 
-export const EntityStepCardBottomSection = ({
-  stepType,
+export const ObjectiveProgressEntity = ({
   formattedEntityData,
-  printVersion,
+  entityCompleted,
+  headingLevel = "h2",
 }: Props) => {
-  switch (stepType) {
-    case OverlayModalStepTypes.OBJECTIVE_PROGRESS:
-      return (
+  return (
+    <>
+      <Heading as={headingLevel} sx={sx.mainHeading}>
+        {formattedEntityData.objectiveName}
+      </Heading>
+      <Text sx={sx.subtitle}>
+        Performance measure description or indicators your state or territory
+        will use to monitor progress towards achievement
+      </Text>
+      <Text sx={sx.description}>{formattedEntityData.description}</Text>
+      <Text sx={sx.subtitle}>Performance measure targets</Text>
+      <Text sx={sx.description}>{formattedEntityData.targets}</Text>
+      {formattedEntityData.quarterProjections.length > 0 && !entityCompleted && (
         <>
-          {(formattedEntityData?.performanceMeasureProgress ||
-            printVersion) && (
+          <Text sx={sx.subtitle}>
+            Quantitative targets for this reporting period
+          </Text>
+          <Grid sx={sx.sarGrid}>
+            {formattedEntityData?.quarterProjections.map((quarter: any) => {
+              return (
+                <GridItem key={quarter.id}>
+                  <Flex sx={sx.gridItems}>
+                    <Text sx={sx.gridSubtitle}>{quarter.id} Target:</Text>
+                    <Text sx={sx.subtext}>{quarter.value}</Text>
+                  </Flex>
+                </GridItem>
+              );
+            })}
+          </Grid>
+        </>
+      )}
+      {entityCompleted ? (
+        <>
+          {formattedEntityData?.performanceMeasureProgress && (
```

Comment:
`formattedEntityData` is always defined so `formattedEntityData?` shouldn't be needed anywhere in this file.